.drone.yml

```diff
@@ -0,0 +1,96 @@
+kind: pipeline
+name: test
+
+steps:
+- name: docker build
+  image: docker:dind
+  volumes:
+  - name: dockersock
+    path: /var/run
+  commands:
+  - sleep 5; until docker version; do sleep 5; done
```

Comment:
How long does it take docker to start?!

Reply:
Between 3 and 10 seconds, but I don't want intermittent errors.

Reply:
Then how about `sleep 1`? We might get some spam in the logs but that's ok.

Reply:
bump :)